linkerd/docs/routers.md

```diff
@@ -207,3 +200,82 @@ minSize | `0` | The minimum number of connections to maintain to each host.
 maxSize | Int.MaxValue | The maximum number of connections to maintain to each host.
 idleTimeMs | forever | The amount of idle time for which a connection is cached in milliseconds.
 maxWaiters | Int.MaxValue | The maximum number of connection requests that are queued when the connection concurrency exceeds maxSize.
+
+<a name="service-configuration"></a>
```

Comment:
This anchor isn't necessary since it matches the h2 header exactly. Same goes for the rest of the headers in this file. Can you remove all of them?